src/lib/client.js

```diff
@@ -94,7 +94,7 @@ export default class Client {
   }
 
   get_apply_now(type) {
-    const url = pdpHost(marketingUI, this.platform) + '/apply.html?' + this.key + '&type=' + type;
+    const url = pdpHost(marketingUI, this.platform) + '/apply.html?public_key=' + this.key + '&type=' + type;
```

Comment:
@cjdeg this was a potentially big problem... this was causing the urlParams detection in ck-marketing to never see a public_key on apply nows since there was no url param named.  This may partly explain why we see so many unattributed apply now flows (i.e. ones that get assigned to the default credit key merchant)